src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
This is a failing test. We need to fix this

Reply:
in the same PR or open separate issue for this?

Reply:
This is just given by the boilerplate. Should I just remove it?

Reply:
The 3 App files here are just boilerplates. Shall I remove them?

<img width="191" alt="Screen Shot 2020-06-04 at 8 08 32 pm" src="https://user-images.githubusercontent.com/29667122/83744329-4b350c00-a69f-11ea-847d-7975a3d54a44.png">

Reply:
Yes if you are not using them, remove it.

Reply:
Got it. All removed now. Anything else I need to do before I pushed the latest commit?